src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -32,40 +33,29 @@ public LineService(LineRepository lineRepository, StationService stationService)
     public LineResponse saveLine(LineRequest request) {
         Station upStation = stationService.findById(request.getUpStationId());
         Station downStation = stationService.findById(request.getDownStationId());
-        Line persistLine = lineRepository.save(new Line(request.getName(), request.getColor(), upStation, downStation, request.getDistance()));
-        List<StationResponse> stations = getStations(persistLine).stream()
-                .map(it -> StationResponse.of(it))
-                .collect(Collectors.toList());
-        return LineResponse.of(persistLine, stations);
+        Line persistLine = lineRepository.save(request.toLine(upStation, downStation));
+        return LineResponse.of(persistLine);
     }
 
     public List<LineResponse> findLines() {
         List<Line> persistLines = lineRepository.findAll();
         return persistLines.stream()
-                .map(line -> {
-                    List<StationResponse> stations = getStations(line).stream()
-                            .map(it -> StationResponse.of(it))
-                            .collect(Collectors.toList());
-                    return LineResponse.of(line, stations);
-                })
+                .map(line -> { return LineResponse.of(line);})
```

Comment:
```suggestion
                .map(line -> LineResponse::of)
```

아래 두 가지 조건을 충족하면 람다 메소드 참조로 대체가 가능합니다.
1. 한 줄이라 return 생략 가능
2. 매개변수가 하나이고 그 매개변수가 그대로 사용

Reply:
확실히 메소드 참조를 활용하니 코드가 더 간결해지네요. 👍
아래 피드백 주신 항목도 포함하여 간결하게 할 수 있는 항목들은 찾아서 적용했습니다.
감사합니다.